webapp/setup.py

```diff
@@ -1,14 +1,14 @@
 from setuptools import find_packages, setup
 
 setup(
-    name="pi_server",
+    name="webapp",
     version="0.1.0",
     packages=find_packages(exclude=["tests"]),
     install_requires="streamlit==1.7.0",
-    extras_require={"dev": ["pytest", "mypy", "black", "isort"]},
+    extras_require={"dev": ["pytest", "mypy", "black"]},
     entry_points={
         "console_scripts": [
-            "run_server = app.__main__:main",
+            "run_server = pi_server.__main__:main",
```

Comment:
pi_server shouldn't be here.

Reply:
@Saxamos  is there because you renamed the app folder to pi_server. 
Do you want me to rename the folder?    app??

Reply:
It's on webapp side here so name it accordingly, else it will not work.

Reply:
ok . solved